src/components/SingleCarousel.module.css

```diff
@@ -2,7 +2,20 @@
   padding: 35px 0;
 }
 
+.class {
```

Comment:
that is a super-vague, kinda lame classname.   making the class .flex would have more worth than calling it just .class...  😆

Reply:
Yeah. I'm surprised a class name of "class" even works without issue.
We have a helpers css file that has some commonly used styling...

```css
.flex {
  display: flex;
}

.flex-wrap {
  display: flex;
  flex-wrap: wrap;
}

.text-muted {
  font-size: .75em;
  color: #999;
}

.section-separator {
  margin: 40px 0;
}

...etc...
```